implementation/jobberknoll/deno.json

```diff
@@ -1,5 +1,5 @@
 {
-  "version": "0.2.0",
+  "version": "0.3.0",
```

Comment:
**thought**: It's not deployed yet, why do you bother?

Reply:
No huge benefits honestly. It lets me see which version I'm running in both the console and both OpenAPI UIs, which isn't really important, but saved me once. And whenever this will actually communicate with other services (starting with Feather which I also have to write :c) it will get more useful.